usecases/steg_asymptotic_2sensors/test_optimize.py

```diff
@@ -0,0 +1,129 @@
+import unittest
+import numpy as np
+from scipy.optimize import least_squares
+from pathlib import Path
+import yaml
+from asymptotic_model_error import AsymptoticModelError
+from bayes.multi_model_error import MultiModelError
+
+
+class MultiAsymptoticModelError(MultiModelError):
+    """
+    Define a class that combines multiple asymptotic model errors
+    """
+    def __init__(self, yaml_file_metadata, yaml_file_data):
+        """Create class combining different asymptotic model errors
+
+        Note:
+            Currently implemented as a class to later add individual routines of adding additional errors
+            e.g. using database queries, etc. currently, everything is only in yaml files and thus in the
+            constructor.
+
+        Args:
+            yaml_file_list(list of str): list of yaml files with the experimental data
+        """
+        super().__init__()
+
+        with open(yaml_file_metadata, "r") as f:
+            meta_data = yaml.load(f, Loader=yaml.FullLoader)
+
+        x_Mat_sensors = np.asarray(meta_data['x_Mat_sensors'])
+        # all_a = np.asarray(meta_data['all_a'])
+
+        with open(yaml_file_data, "r") as f:
+            data = yaml.load(f, Loader=yaml.FullLoader)
+
+#        for idx, a in enumerate(all_a):
+        data_MatB = np.asarray(data['MatB'])
+        data_MatC = np.asarray(data['MatC'])
+        data_Mat_B_C = np.asarray(data['Mat_B_C'])
+        # assert(a == data['a'])
+        # assert(b == data['b'])
+        # assert(c == data['c'])
+        # assert(d == data['d'])
+        # assert(e == data['e'])
+        single_model_error = AsymptoticModelError(x_Mat_sensors, data_MatB, data_MatC, data_Mat_B_C)
+            # read in the parameters that are given in the experiment (in this case constant offset a)
+        parameter = single_model_error.get_parameter_dict()
+            # add parameters that are model parameters (not given in the experimental data)
+        parameter.define("mA")
+        parameter.define("mB") #! was b
+        self.add(single_model_error, parameter)
+
+        # define one shared latent parameter (only one parameter for all models)
+        self.latent.add_by_name('mA')
+#        self.latent.add_by_name('mB')

```

Comment:
so do you assume mB to be known? In this case, you would have to define a value to mB at some point.